### PR TITLE
Add dashboard for ErlangVM

### PR DESCRIPTION
**Pull request by Fabs**
Adding Graphs for the Erlang VM. You can this dashboard on soak113s.

image

### CODE REVIEW

**Comment by urbanserj** [comment]
@Fabs could you please remove `Erlang` from `Networking: Erlang VM`? it will just cause more questions, nobody cares while they don't know what stack we are using.

**Comment by Fabs** [comment]
@urbanserj done.

File: dashboards/Networking/Networking-VM.json [hunk 1]
```diff
@@ -0,0 +1,1397 @@
...
+          "show": true
+        }
+      ],
+      "yaxis": {
+        "align": false,
+        "alignLevel": null
+      }
+    },
+    {
+      "content": "This are the set of panels about VM CPU consumption.\n \n**VM GC**: Shows the number of GCs performmed and how many bytes they reclaimed. With multiple hosts show the `max` bytes reclaimed and the `max` number of GC among the hosts.\n\n**Queues Length**: Shows the queue length of the selected host.\n\n**Load**: shows the number of Reductions and Context Switches performmed by the [VM scheduler](https://hamidreza-s.github.io/erlang/scheduling/real-time/preemptive/migration/2016/02/09/erlang-scheduler-details.html).\n\n**Processes**: shows total number of processes and ports ready to run at any given run-queue, the total number of process current running at the node and the limit in number of processes when set.",
```
Comment (philipnrmn) [inline dashboards/Networking/Networking-VM.json]:
```suggestion
      "content": "This set of panels shows VM CPU consumption.\n \n**VM GC**: Shows the number of GCs performed and how many bytes they reclaimed. With multiple hosts, shows the `max` bytes reclaimed and the `max` number of GC among the hosts.\n\n**Queues Length**: Shows the queue length of the selected host.\n\n**Load**: shows the number of Reductions and Context Switches performed by the [VM scheduler](https://hamidreza-s.github.io/erlang/scheduling/real-time/preemptive/migration/2016/02/09/erlang-scheduler-details.html).\n\n**Processes**: shows the total number of processes and ports ready to run at any given run-queue, the total number of process current running at the node and the limit in number of processes when set.",
```

File: dashboards/Networking/Networking-VM.json [hunk 2]
```diff
@@ -0,0 +1,1397 @@
...
+      "6h",
+      "12h",
+      "24h",
+      "2d",
+      "7d",
+      "30d"
+    ]
+  },
+  "timezone": "browser",
+  title": "Networking: VM",
```
Comment (philipnrmn) [inline dashboards/Networking/Networking-VM.json]:
```suggestion
  "title": "Networking: VM",
```

File: dashboards/Networking/Networking-VM.json [hunk 3]
```diff
@@ -0,0 +1,1397 @@
...
+          "show": false
+        }
+      ],
+      "yaxis": {
+        "align": false,
+        "alignLevel": null
+      }
+    },
+    {
+      "content": "This are the set of panels about Uptime.\n \n**Uptime**: This host uptime in hours.\n\n**Uptime: Longest Running VMs**: Longest living VMs based on top 5 uptime.\n\n**Uptime: Most Recent Restarts**: Most Recent Restarts based on bottom 5 uptime.",
```
Comment (philipnrmn) [inline dashboards/Networking/Networking-VM.json]:
```suggestion
      "content": "This set of panels shows uptime.\n \n**Uptime**: This host's uptime in hours.\n\n**Uptime: Longest Running VMs**: Longest living VMs based on top 5 uptime.\n\n**Uptime: Most Recent Restarts**: Most recent restarts based on bottom 5 uptime.",
```

File: dashboards/Networking/Networking-VM.json [hunk 4]
```diff
@@ -0,0 +1,1397 @@
...
+          "show": true
+        }
+      ],
+      "yaxis": {
+        "align": false,
+        "alignLevel": null
+      }
+    },
+    {
+      "content": "This are the set of panels about VM Memory consumption.\n\n**VM Memory**: This graph shows the total memory used by the system + `processes`. The `system` memory is further broken down according to the type of usage `atoms`, `binary`, `code` and `ets` tables.",
```
Comment (philipnrmn) [inline dashboards/Networking/Networking-VM.json]:
```suggestion
      "content": "This set of panels shows VM Memory consumption.\n\n**VM Memory**: This graph shows the total memory used by the system + `processes`. The `system` memory is further broken down according to the type of usage `atoms`, `binary`, `code` and `ets` tables.",
```

File: dashboards/Networking/Networking-VM.json [hunk 5]
```diff
@@ -0,0 +1,1397 @@
...
+          "show": true
+        }
+      ],
+      "yaxis": {
+        "align": false,
+        "alignLevel": null
+      }
+    },
+    {
+      "content": "This are the set of panels about VM I/O.\n\n**VM IO**: rate of bytes transfered in and out the VM of the selected Host.\n\n**Ports**: the VM total open [Ports](http://erlang.org/doc/reference_manual/ports.html) as well as the max ports available (dotted red line) if defined.",
```
Comment (philipnrmn) [inline dashboards/Networking/Networking-VM.json]:
```suggestion
      "content": "This set of panels shows VM I/O.\n\n**VM IO**: rate of bytes transferred in and out the VM of the selected Host.\n\n**Ports**: the VM total open [ports](http://erlang.org/doc/reference_manual/ports.html) as well as the max ports available (dotted red line) if defined.",
```

**Review by philipnrmn — CHANGES_REQUESTED** [review]
Nice work Fabs! Especially loving the helpful descriptions.

I have some minor grammar and spelling fixes, and also there's a missing quote which invalidates the JSON at the bottom.

File: dashboards/Networking/Networking-VM.json (hunk 2 above)
Comment (Fabs) [reply dashboards/Networking/Networking-VM.json]: Oh a lot, thanks for that one :)

**Comment by Fabs** [comment]
@philipnrmn, thanks a lot for the review, could you please look at it again?

**Review by philipnrmn — APPROVED** [review]

**Pull request merged**